packages/react/src/lib/accessibility/Behaviors/Alert/alertWarningBehavior.ts

```diff
@@ -1,17 +1,21 @@
 import { Accessibility } from '../../types'
+import AlertBehaviorProps from './alertBehaviorProps'
 
 /**
  * @specification
- * Adds role 'alert' to 'content' slot.
- * Adds attribute 'aria-live=polite' to 'content' slot.
+ * Adds role 'alert' to 'wrapper' element.
```

Comment:
please keep 'slot' here as it is a slot from the behavior point of view (I understand it is not a slot in the component). This will also make the unit tests work again.